### ISSUE TITLE
append() and prepend() are not supported.

### ISSUE THREAD

**Issue by wout** [svgdotjs/svgdom#98]
I'm using svgdom with without SVG.js and my code failed because it uses `Element.append()` and `Element.prepend()`. the error I'm getting is:

```
file:///app/node_modules/svgdom/src/utils/nodesToNode.js:3
  nodes = nodes.map((node) => {
                ^

TypeError: nodes.map is not a function
    at nodesToNode (file:///app/node_modules/svgdom/src/utils/nodesToNode.js:3:17)
    at SVGSVGElement.prepend (file:///app/node_modules/svgdom/src/dom/mixins/ParentNode.js:34:18)
    at file:///app/index.js:16:5
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
```

If I use `appendChild()` instead, it does work.

**Comment by Fuzzyma** [comment]
Looks like this api is not implemented yet. Will add it if I find some time :).

Feel free to create a PR to add it yourself 😄

**Comment by Fuzzyma** [comment]
nvm, it is implemented. if nodes.map is not a function, what did you pass into the append fn? Isnt it an array?

**Comment by Fuzzyma** [comment]
fixed in [0.1.12](https://github.com/svgdotjs/svgdom/releases/tag/0.1.12)

**Comment by wout** [comment]
Brilliant, thanks! :)

**Comment by wout** [comment]
There still seems to be an issue. The error has gone, but the objects are not inserted into the DOM with `append()` and `prepend()`.

**Comment by Fuzzyma** [comment]
Gnaa thought it was a quick fix. Alright let me _actually_ look into this 😄

**Comment by Fuzzyma** [comment]
alright, this time i added tests :D

fixed in [0.1.13](https://github.com/svgdotjs/svgdom/releases/tag/0.1.13)

**Comment by wout** [comment]
Thanks again! :smile: 
I'll test it out later today!
